streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -536,6 +537,11 @@ public class StreamsConfig extends AbstractConfig {
     public static final String ROCKS_DB = "rocksDB";
     public static final String IN_MEMORY = "in_memory";
 
+    /** {@code dsl.store.suppliers.class } */
+    public static final String DSL_STORE_SUPPLIERS_CLASS_CONFIG = "dsl.store.suppliers.class";
+    public static final String DSL_STORE_SUPPLIERS_CLASS_DOC = "Defines which store implementations to plug in to DSL operators. Must implement the <code>org.apache.kafka.streams.state.DslStoreSuppliers</code> interface.";
```

Comment:
nit: the doc string and default can (and probably should) just be `private`. Not a big deal but it keeps them out of the official public API and helps avoid awkward/annoying trivial things like having to deprecate all of these as well, not being able to move/rename if necessary,  you get the idea

Reply:
kept package private so it can be used in `TopologyConfig` as well